custom/icds_reports/utils/__init__.py

```diff
@@ -1101,22 +1101,35 @@ def create_thr_report_excel_file(excel_data, data_type, month, aggregation_level
     bold_font = Font(bold=True)
     blue_fill = PatternFill("solid", fgColor="B3C5E5")
     grey_fill = PatternFill("solid", fgColor="BFBFBF")
+    purple_fill = PatternFill("solid", fgColor="CFC6F5")
 
     workbook = Workbook()
     worksheet = workbook.active
     # sheet title
     worksheet.title = "THR Report"
     worksheet.sheet_view.showGridLines = False
-    amount_of_columns = 11 - aggregation_level
-    last_column = string.ascii_uppercase[amount_of_columns]
-    worksheet.merge_cells('B2:{0}2'.format(last_column))
-    title_cell = worksheet['B2']
+
+    if report_type == 'days_beneficiary_wise':
+        total_column_count = 30
+        data_start_row_diff = 3
+    elif report_type == 'beneficiary_wise':
+        total_column_count = 15
+        data_start_row_diff = 2
+    else:
+        total_column_count = 11
+        data_start_row_diff = 1
```

Comment:
hey yeah, I think we can find that with the length of the headers list. But I thought it will not be very clear why are these different. And this report type parameter is anyway being supplied from FE in THR report. and each report type "technically" corresponds to a different report generated with different columns. 
Thats why I though to use the report_type for this so that its clears which setting is for what